app/src/main/res/values/strings.xml

```diff
@@ -285,16 +285,20 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
     <string name="quest_sport_roller_skating">Roller Skating</string>
     <string name="quest_wheelchairAccess_name_title">Is “%s” wheelchair accessible?</string>
     <string name="quest_wheelchairAccess_bus_station_title">Is this bus station wheelchair accessible?</string>
+    <string name="quest_wheelchairAccess_toilets_title">Are these toilets wheelchair accessible?</string>
     <string name="quest_wheelchairAccess_subway_entrance_title">Is this subway entrance wheelchair accessible?</string>
     <string name="quest_wheelchairAccess_railway_station_title">Is this railway station wheelchair accessible?</string>
     <string name="quest_wheelchairAccess_location_title">Is this location wheelchair accessible?</string>
     <string name="quest_wheelchairAccess_bus_station_name_title">Is the bus station “%s” wheelchair accessible?</string>
+    <string name="quest_wheelchairAccess_toilets_name_title">Are the toilets “%s” wheelchair accessible?</string>
```

Comment:
Do toilets have names?

Reply:
Yes of course, in large cities they have names so that they can be referred to easily (by the people and maintenance team).